content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
+The `timestamp` attribute signifies the time at which the message was generated by its sender. This field SHOULD contain the Unix epoch time in nanoseconds. If the attribute is omitted, it SHOULD be interpreted as timestamp 0.
 
-## Payloads
+The `ephemeral` attribute signifies the transient nature of the message. If this field is set to `true`, the message SHOULD be interpreted as ephemeral. Else, if the field is omitted or set to `false`, the message SHOULD be interpreted as non-ephemeral.
 
-Payloads are implemented using [protocol buffers v3](https://developers.google.com/protocol-buffers/).
+# Wire Format
+
+The `WakuMessage` wire format is specified using [protocol buffers v3](https://developers.google.com/protocol-buffers/).
 
 ```protobuf
 syntax = "proto3";
 
 message WakuMessage {
   bytes payload = 1;
-  string contentTopic = 2;
+  string content_topic = 2;
   uint32 version = 3;
   sint64 timestamp = 10;
   bool ephemeral = 31;
 }
 ```
 
-## Payload encryption
-
-Payload encryption depends on the `version` field.
-
-### Version 0
+# Payload encryption
 
-This indicates that the payload SHOULD be either unencrypted or that encryption is done at a separate layer outside of Waku.
+The waku message payload MAY be encrypted. The message `version` attribute indicates the schema used to encrypt the payload data.
 
-### Version 1
+- **Version 0:**  The payload SHOULD be interpreted as unencrypted; additionally, it CAN indicate that the message payload has been encrypted at the application layer.
+- **Version 1:** The payload SHOULD be encrypted using Waku v1 payload encryption. [26/WAKU-PAYLOAD](/spec/26).
+This provides asymmetric and symmetric encryption. The key agreement is performed out of band. And provides an encrypted signature and padding for some form of unlinkability.
+- **Version 2:** The payload SHOULD be encrypted using [35/WAKU2-NOISE](/spec/35).
+Waku Noise protocol provides symmetric encryption and asymmetric key exchange.
 
-This indicates that payloads MUST be encrypted using [WAKU2-PAYLOAD](/spec/26).
+Any `version` value not included in the list is reserved for future use, and the payload SHOULD be interpreted as unencrypted.
 
-This provides for asymmetric and symmetric encryption.
-Key agreement is out of band.
-It also provides an encrypted signature and padding for some form of unlinkability.
+# Security Considerations
 
-### Version 2
+## Confidentiality, integrity, and authenticity
 
-This indicates that payloads MUST be encoded using [35/WAKU-NOISE](/spec/35).
+The level of confidentiality, integrity, and authenticity of the waku message payload is discretionary. Accordingly, the application layer shall utilize the encryption and signature schemes supported by Waku v2 to meet the application-specific privacy needs.
 
-This provides for symmetric encryption and
-asymmetric key-exchange protocols.
+## Reliability of the `timestamp` attribute
 
-# Differences from Whisper / Waku v1 envelopes
```

Comment:
Is this intentional to remove this section? curious to know why

Reply:
IMHO, there is little value in comparing the Waku v2 message format with whisper/waku v1 envelope formats for this RFC. But it may be relevant in another RFC like the waku v1 to waku v2 bridging RFC.

We can restore it if we think it adds some value. Otherwise, I'd rather keep the RFC succinct.

Reply:
@oskarth How relevant is this "Whisper/Waku V1 comparison section"?